### PR TITLE
Update Enable CIM on AWS in 2.6

### PR DESCRIPTION
**Pull request by oafischer**
https://github.com/stolostron/backlog/issues/26095

### CODE REVIEW

**Comment by jelkosz** [comment]
lgtm

File: multicluster_engine/cluster_lifecycle/create_infra_env.adoc [hunk 1]
```diff
@@ -136,9 +136,18 @@ Your CIM service is configured. You can verify that it is healthy by checking th
...
-If you are running CIM on Amazon Web Services, complete the following additional steps after  <<enable-cim,Enabling CIM>>:
+If you are running your hub on Amazon Web Services and want to enable the CIM service, complete the following additional steps after  <<enable-cim,Enabling CIM>>:
 
-. Create a new `IngressController` with a unique domain using the `NLB` `type` parameter. See the following example:
+. Find the unique domain configured on the `assisted-image-service` by running the following command:
++
+----
+oc get routes --all-namespaces | grep assisted-image-service
+----
++
```
Comment (berenss) [inline multicluster_engine/cluster_lifecycle/create_infra_env.adoc:146]: to do the above step, you already need to " Make sure you are logged in at the hub"

**Review by berenss — APPROVED** [review]
Nice work thank you!

File: multicluster_engine/cluster_lifecycle/create_infra_env.adoc (hunk 1 above)
Comment (oafischer) [reply multicluster_engine/cluster_lifecycle/create_infra_env.adoc:146]: Added it, thanks!

File: multicluster_engine/cluster_lifecycle/create_infra_env.adoc [hunk 2]
```diff
@@ -136,9 +136,18 @@ Your CIM service is configured. You can verify that it is healthy by checking th
 [#enable-cim-aws]
 === Enabling Central Infrastructure Management on Amazon Web Services
 
-If you are running CIM on Amazon Web Services, complete the following additional steps after  <<enable-cim,Enabling CIM>>:
+If you are running your hub on Amazon Web Services and want to enable the CIM service, complete the following additional steps after  <<enable-cim,Enabling CIM>>:
```
Comment (dockerymick) [inline multicluster_engine/cluster_lifecycle/create_infra_env.adoc]: I recommend changing "hub" to "hub cluster"

File: multicluster_engine/cluster_lifecycle/create_infra_env.adoc [hunk 3]
```diff
@@ -162,35 +171,48 @@ spec:
...
-
-. Add `<yourdomain>` to the `domain` parameter in `IngressController` by replacing `<domain>` in `nlb-apps.<domain>.com` with `<yourdomain>`.
 
 . Run the following command to edit the `assisted-image-service` route to use the `nlb-apps` location:
 +
 ----
 oc edit route assisted-image-service -n <namespace>
 ----
++
+*Tip:* The default namespace is where you installed the multicluster engine.
```
Comment (dockerymick) [inline multicluster_engine/cluster_lifecycle/create_infra_env.adoc]: I recommend that you use the conref for multicluster engine

**Review by dockerymick — APPROVED** [review]
Made some suggestions

**Pull request merged**